### PR TITLE
feat: Update to C# 12.0

### PR DESCRIPTION
**Pull request by BenjaminMichaelis**

### CODE REVIEW

File: IntelliTect.Multitool/ReleaseDateAttribute.cs [hunk 1]
```diff
@@ -6,22 +6,17 @@ namespace IntelliTect.Multitool;
 /// <summary>
 /// The release date assembly attribute
 /// </summary>
+/// <remarks>
```
Comment (Keboo) [inline IntelliTect.Multitool/ReleaseDateAttribute.cs:9]: Curious why switch away from summary?

**Review by Keboo — APPROVED** [review]

File: IntelliTect.Multitool/ReleaseDateAttribute.cs (hunk 1 above)
Comment (BenjaminMichaelis) [reply IntelliTect.Multitool/ReleaseDateAttribute.cs:9]: This is just the way the refactoring to primary constructor transferred the comments, I noticed it but assumed that it knew what the standard for the commenting on primary constructors is going to be so I left it, thoughts?

Comment (Keboo) [reply IntelliTect.Multitool/ReleaseDateAttribute.cs:9]: Seems like a bug in the refactoring IMO.
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags?WT.mc_id=DT-MVP-5003472#general-tags

Comment (BenjaminMichaelis) [reply IntelliTect.Multitool/ReleaseDateAttribute.cs]: I'll remove, https://github.com/IntelliTect/Multitool/pull/101/commits/cd4a0799672098817cf2462ebd29367279ca8153

If I should change it again just lmk and I'll create a patch

**Pull request merged**